skills.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
I see the json which i have to add in the mongodb has changes, like you have added 
keywords alongside name and level. Then why doesn't this have that? which is the single source of truth?

Reply:
This should also have the things, I will push the changes.